src/controllers/hid/hidenumerator.cpp

```diff
@@ -11,6 +11,10 @@ bool recognizeDevice(const hid_device_info& device_info) {
     // Skip mice and keyboards. Users can accidentally disable their mouse
     // and/or keyboard by enabling them as HID controllers in Mixxx.
     // https://bugs.launchpad.net/mixxx/+bug/1940599
+    // On Linux usage_page and usage are always 0
+    // Reading usage_page and usage is intentionally disabled in hid.c
+    // because it is very invasive as it requires the detach and re-attach
+    // of the kernel driver
```

Comment:
I think, that this is only the case for the libusb backend of hidapi, which was used until 2.3 for Linux. This should work with Main, where the native Linux Kernel backend hidraw is used instead.

Reply:
I think so, only libusb backend is missing for this functionality:
https://github.com/mixxxdj/mixxx/blob/b2401b9b0f14b68163bd580490dfcbf5af57d650/lib/hidapi/hidapi/hidapi.h#L98-L103

Reply:
There  is nothing to do in main, the mouse is there rejected as desired. A merge with main will conflict and we can remove this commit when resolving the conflict.